docs/blog/august-contributions-showcase.md

```diff
@@ -0,0 +1,134 @@
+---
+title: August contributions showcase
+author:
+  - name: James Garbutt
+sidebar: false
+date: 2024-09-07
+head:
+  - - meta
+    - property: og:type
+      content: website
+  - - meta
+    - property: og:title
+      content: August contributions showcase
+  - - meta
+    - property: og:url
+      content: https://e18e.dev/blog/august-contributions-showcase
+  - - meta
+    - property: og:description
+      content: A showcase of e18e-focused contributions from August
+---
+
+# August contributions
+
+_September 7, 2024_
+
+We've seen another great month of contributions from the community :tada: Here's just a few highlights.
+
+If you want to get involved, remember to come [join the discussion](https://chat.e18e.dev)!
+
```

Comment:
maybe we could add a heading level to differentiate libraries to be aware of vs packages that are getting cleaned up
```
# Libraries to be aware of

## tinyexec
...

## tinyglobby / fdir
...
...

# Package cleanup efforts

## changesets
...
```

Reply:
Good idea. I'll shuffle things around 👍